index.html

```diff
@@ -31,7 +31,139 @@ <h1>I’m Mariane
      <li><a href=""><img src="images/vector2.svg" alt=""></a></li>
     </ul>
    </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot.png" alt=""></article>
+    <article>
+     <h1>Tonic</h1>
+     <div class="frame">
+      <h2>CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
```

Comment:
- [ ] To keep your HTML semantic, you should consider the following structure:
```
<ul class="lang">
      <li>html</li>
      <li>css</li>
      <li>javaScript</li>
</ul>
```
> You can read more about unordered lists [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul).
- [ ] Keep in mind that you should also apply the changes to the following articles.